File: audiorev.m
inputFileName='input_audio.wav';
outputFileName='reversed_audio.wav';
[inputData,sampleRate]=audioread("/home/rguktrkvaleey/Downloads/Jingle.wav");
reversedData=flipud(inputData);
audiowrite(outputFileName,reversedData,sampleRate);
disp('Audio file is reversed and saved successfully.');

File: audiorev.py
from pydub import AudioSegment

def reverse_audio(input_file, output_file):
    audio = AudioSegment.from_file(input_file)
    reversed_audio = audio.reverse()
    reversed_audio.export(output_file, format="wav")
if __name__ == "__main__":
    input_file = "/home/rguktrkvaleey/Downloads/Classical.wav"
    output_file = "reversed_audio.wav"
    reverse_audio(input_file, output_file)

